pages/linux/hashcat.md

```diff
@@ -0,0 +1,32 @@
+# hashcat
+
+> Fast and advanced password recovery tool.
+> More information: <https://hashcat.net/hashcat/>.
+
+- Brute-force attack with default hashcat mask:
+
+`hashcat --hash-type {{id_hash_type}} --attack-mode 3 {{hash}}`
+
+- Brute-force attack with a known pattern of 4 digits:
+
+`hashcat --hash-type {{id_hash_type}} --attack-mode 3 {{hash}} ?d?d?d?d`
+
+- Brute-force attack with max length of 8 chars using the complete charset (numbers + letters + special chars):
+
+`hashcat --hash-type {{id_hash_type}} --attack-mode 3 --increment {{hash}} ?a?a?a?a?a?a?a?a`
+
+- Dictionary attack using the RockYou wordlist of a Kali Linux box:
+
+`hashcat --hash-type {{id_hash_type}} --attack-mode 0 {{hash}} /usr/share/wordlists/rockyou.txt`
```

Comment:
```suggestion
`hashcat --hash-type {{id_hash_type}} --attack-mode 0 {{hash}} {{/usr/share/wordlists/rockyou.txt}}`
```
Do all the paths have to be absolute? If not please remove the leading slashes everywhere.

Reply:
the path is absolute and standard when using Kali linux

Reply:
i added the brackets